packages/calypso-products/src/types.ts

```diff
@@ -224,7 +226,7 @@ export type StorageOption = {
 
 export type Plan = BillingTerm & {
 	group: typeof GROUP_WPCOM | typeof GROUP_JETPACK;
-	type: string;
+	type: PlanType;
```

Comment:
I changed it to a union type of the `TYPES_LIST`. Nothing broke, and we have more type safety now. I wonder why it wasn't like that before?

Reply:
Nice. 

> I wonder why it wasn't like that before?

No idea. A lot of things weren't done from before :D